src/main/java/aeneas/models/PuzzleLevel.java

```diff
@@ -11,10 +11,12 @@
   
   public PuzzleLevel(Bullpen bullpen) {
     super(bullpen);
+    board = new PuzzleBoard();
```

Comment:
two constructors is acceptable. Make sure you note in javadocs how each one behaves.